app/src/main/java/com/ntg/user/mvpsample/TaskPresenter.java

```diff
@@ -0,0 +1,61 @@
+package com.ntg.user.mvpsample;
+
+
+import com.ntg.user.mvpsample.remote.Task;
+
+import java.util.List;
+
+/**
+ * Created by GM7 on 1/29/2018.
+ */
```

Comment:
we want to make for each module one presenter / one model 
show tasks view , show tasks presenter / show tasks repo
save task view , save task presenter / save task repo
show task details view /  show task details  presenter  / no need for repo as task object already passed.